eland/tests/ml/test_imported_ml_model_pytest.py

```diff
@@ -322,3 +333,35 @@ def test_predict_single_feature_vector(self):
 
         # Clean up
         es_model.delete_model()
+
+    @requires_lightgbm
+    @pytest.mark.parametrize("compress_model_definition", [True, False])
+    def test_lgbm_regressor(self, compress_model_definition):
```

Comment:
I think this ought really to be testing `LGBMClassifier` as well.

Reply:
@tveasey support for that class will happen later. This PR is regression only (to keep size small)

Reply:
And indeed you haven't implemented the sigmoid transforms. Good point!